tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -15,12 +15,15 @@ details: |
     list and deletion.
 
 prepare: |
-    echo "Given openvswitch is installed"
-    apt install -y --no-install-recommends openvswitch-switch
+    # Note that the order of the installs is to avoid a timeout issue on the 
```

Comment:
It is not a problem if a snap is installed before, it is just giving time, as I understand after the reset many services are starting and doing their stuff and it could the the problem. In the past there were issues opened because of this problem, the most common was that there was a deadlock trying to access to some shared resouces. So this change is a way to reduce the propability that this happen again.